### PR TITLE
Anbonik dev

### PR DESCRIPTION
**Pull request by anbonik**

### CODE REVIEW

**Review by Maxoo005 — COMMENTED** [review]
Siemka, rozwiązanie solidne, prezentuje sie świetnie i spełnia wszelkie wymagania

**Review by nuklearbanana — APPROVED** [review]
Dogłębnie przeanalizowana sieć neuronowa z liniowymi warstwami i trafne wnioski.  
Tak wysoką dokładność w tym przypadku może powodować długi czas treningu i dużo epok co nie jest złe, ale można się zastanowić czy tracenie dużej ilości zasobów i czasu na tak długim szkoleniu sieci z warstw liniowych jest sensowne w przypadku gdy po wytrenowaniu sieci z warstwami konwolucyjnymi otrzymuje się zbliżony efekt po zaledwie 2 epokach. 

Jedyne co bym poprawił to czytelność wyników i zwizualizowania ich w jakiś sposób.

**Review by lolakk05 — COMMENTED** [review]
Solidna praca i analiza tego co właściwie się dzieje z modelami, pomijając że zrobiłeś dla samych zwykłych sieci liniowych a nie dałeś ani razu CNN (chyba, że jestem ślepy to przepraszam) i troszkę brakuje chociaż jakiegoś wykresiku

**Review by lolakk05 — APPROVED** [review]

**Review by Tczek — APPROVED** [review]
Dobrze to tak brakuje mi tego CNN i jeszcze brakuje mi więcej insightu na prace modelu bo samo accuracy mało nam mówi jak poszczególne rozpoznawanie cyfr wygląda

**Comment by Karo555** [comment]
Hej,
poniżej sugestie.
1) brak pre-processingu danych
2) szkoda, że nie pojawił się ani razu CNN
3) etykiet nie spłaszczamy tzn.
`y_train_flat = x_train.reshape(x_train.shape[0], -1)`
`y_test_flat = x_test.reshape(x_test.shape[0], -1)`
jest błędem
4) mieszasz TensorFlow z PyTorch - niby można, ale po co? ;--) albo jedno albo drugie
